vector/src/main/res/xml/riotx_provider_paths.xml

```diff
@@ -3,4 +3,9 @@
     <cache-path
         name="shared"
         path="/" />
+
+    <external-path
+        name="external_files"
```

Comment:
Is it mandatory to add this? I do not think so, as it is to export file from RiotX which is not the subject of this PR

Reply:
Actually, it was added for handling photo capture, but it's now handled by the lib, so can remove yup.